assets/js/components/Dashboard/PaMessageForm/PaMessageForm.tsx

```diff
@@ -101,55 +135,57 @@ const PaMessageForm = ({
 
   return (
     <div className="new-pa-message">
-      {page === Page.MAIN && (
-        <MainForm
-          onSubmit={() => {
-            const formData: PaMessageFormData = {
-              alert_id: associatedAlert.id,
-              start_datetime: startDateTime.toISOString(),
-              end_datetime: endWithEffectPeriod
-                ? null
-                : endDateTime.toISOString(),
-              days_of_week: days,
-              sign_ids: signIds,
-              priority,
-              interval_in_minutes: Number(interval),
-              visual_text: visualText,
-              audio_text: phoneticText,
-            };
-
-            onSubmit(formData);
-          }}
-          {...{
-            title,
-            days,
-            interval,
-            navigateTo: setPage,
-            phoneticText,
+      <MainForm
+        hide={page !== Page.MAIN}
```

Comment:
Just curious, is there a benefit to hiding the page like this or just preference?